src/css/editor.scss

```diff
@@ -1,50 +1,320 @@
-@import 'common';
+@import 'common/common';
+
+
+$editor-width: 450px;
+$editor-font-size: 14px;
+$editor-std-spacing: 15px;
+
+/* -- PP Adder -- */
+.pp-adder {
+    position: absolute;
+    width: 140px;
+    height: 40px;
+    margin-left: -70px;
+    margin-top: -48px;
+
+    .create-annotation {
+        border-color: #2185D0 !important; // override semantic ui
+        color: #2185D0 !important; // override semantic ui
+        cursor: pointer;
+    }
+
+    .create-annotation:hover {
+        color: darken(#2185D0, 10%) !important; // override semantic ui
+    }
+}
+
 
 /* -- PP Editor -- */
+.pp-editor .pp-widget {
+    width: $editor-width;
+    background-color: white;
+    font-size: $editor-font-size;
+    font-family: $light-font-family;
+}
+
+/* --- Editor head bar --- */
+.pp-editor .pp-editor-head-bar {
+    padding: 0 2*$editor-std-spacing 0 $editor-std-spacing;
+    margin: $editor-std-spacing 0 $editor-std-spacing 0;
+
+    .priority-header {
+        display: inline-block;
+        font-family: $bold-font-family;
+        margin: 0 0 10px 0;
+    }
+    /* priority buttons */
+    .pp-editor-priority {
+        display: inline-block;
+        width: calc(100% - 10px);
+        border: none;
+        outline: none;
+        margin: 0;
+        padding: 3px 11px;
+        height: 3em;
+
+        text-align: center;
+        color: rgb(255, 255, 255);
+        font-family: $bold-font-family;
+        text-transform: uppercase;
+
+        background-color: #D8DADA;
+        cursor: pointer;
+    }
+    /* priority-specific styles */
+    .priority-normal, .priority-warning, .priority-alert {
+        display: inline-block;
```

Comment:
unless we want to support oldschool browsers like IE 6 - IE 10, I'd suggest setting parent of these elements to `display: flex` and let flexbox do the rest of the magic 😉 (instead of `inline-block` and then setting `width` manually etc.)